mission_control/navigator_motion_planner/object_classification.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+from navigator_msgs.msg import PerceptionObject, BuoyArray
+from visualization_msgs.msg import Marker
+from txros import NodeHandle, util
+from twisted.internet import defer, reactor
+
+nh = None
+
+
+class ObjectClassifier:
+
+    def __init__(self):
+        print "init OC"
+        self.classified_ids = {}
+        self.currently_classifying = False
+
+    @util.cancellableInlineCallbacks
+    def _init(self):
+        self.nh = yield NodeHandle.from_argv("object_classifier")
+        global nh
+        nh = self.nh
+
+        self.pub_obj_found = yield self.nh.advertise('/classifier/object', PerceptionObject)
+        self.pub_object_searching = yield self.nh.advertise('/classifier/looking_for', Marker)
```

Comment:
Ooo ok I will try this. It is not super important for right now since this is just a temporary object classification system.